CONTRIBUTING.rst

```diff
@@ -0,0 +1,479 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
CONTRIBUTING.md still exists?

We don't want both. Also do you know if GitHub treats CONTRIBUTING.rst as equally as it does the .md.

For example, when creating a PR this shows up in the right-hand side of the  page:

![Screen Shot 2019-03-21 at 10 06 32](https://user-images.githubusercontent.com/34150/54745355-3450e980-4bc1-11e9-88f7-3a483fa2551c.png)

(That is a link to the .md)

Reply:
https://help.github.com/en/articles/setting-guidelines-for-repository-contributors doesn't mention anything about supported file types, but does mention it could live in`docs/`

Reply:
https://github.com/mlflow/mlflow/compare/master...java-client?expand=1 - looks like .rst is fine!

Reply:
I checked
![Screenshot 2019-03-21 at 11 16 32](https://user-images.githubusercontent.com/12058428/54745848-e80aa700-4bca-11e9-8d16-18233ea8c7d7.png)

Reply:
This looks good. I think we should remove the old UPDATING. I agree with Ash that maintaining both of them isn't a good idea.

Reply:
I also do not want two files. I forgot to delete the old file.